File: flight_most_booked_destinations/v1/get/iOs SDK/FlightMostBookedDestinations.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Flight Most Booked Destinations
amadeus.travel.analytics.airTraffic.booked.get(params: ["originCityCode": "MAD",
                                                        "period": "2017-08"],
                                               onCompletion: {
                                                   response, error in
                                                   if error == nil {
                                                       print(response!.data)
                                                   }
                                        })

File: flight_most_booked_destinations/v1/get/Node SDK/flight_most_booked_destinations.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Where were people flying to from Madrid in the August 2017?
    const response = await amadeus.travel.analytics.airTraffic.booked.get({
      originCityCode: "MAD",
      period: "2017-08",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
